src/debugger/main.ts

```diff
@@ -293,36 +293,44 @@ class RubyDebugSession extends DebugSession {
         return localPath.replace(/\\/g, '/');
     }
 
-	protected convertClientPathToDebugger(localPath: string): string {
```

Comment:
I found that most of this code was formatted with tabs while the rest of the file was using spaces so I fixed that up too!

![spaces](https://thumbs.gfycat.com/RewardingFrigidIrishwaterspaniel-max-1mb.gif)